tests/README.md

```diff
@@ -140,15 +143,10 @@ After the compilation is completed, you can check if the extension is loaded:
 ```
```

Comment:
Can we add some documentation how the new migration models work and how you can add new tables etc.